packages/store-api/src/typeDefs/aggregateOffer.graphql

```diff
@@ -0,0 +1,10 @@
+type StoreAggregateOffer {
```

Comment:
Can you explain why we are using camelcase on filenames?

Reply:
Just because. I think we can change this, no problem. Do you have a preferred one?

Reply:
Just my two cents here, I personally would keep the file names as they are, but would change the name of the folder from `typeDefs` to just `types` or even `graphql`.

Reply:
graphql-tools and apollo, they call these type definitions of `typeDefs`. You can see some examples in [here](https://www.apollographql.com/docs/tutorial/schema/). 

Having that said, I wanted to keep the standard and have a folder called `typeDefs` for the `typeDefs` property of these tools

Reply:
That's great then! Let's stick to the standard 😄